sql/moz-fx-data-shared-prod/telemetry_derived/agg_active_users_v1/metadata.yaml

```diff
@@ -0,0 +1,21 @@
+friendly_name: Agg Active Users
+description: |-
+  Active users aggregated by product and business dimensions.
+  Proposal: https://docs.google.com/document/d/1qvWO49Lr_Z_WErh3I3058A3B1YuiuURx19K3aTdmejM/edit?usp=sharing
+owners:
+- lvargas@mozilla.com
+labels:
+  incremental: true
+scheduling:
+  dag_name: bqetl_analytics_aggregations
+  task_name: agg_active_users
+bigquery:
+  time_partitioning:
+    field: submission_date
+    type: day
+    require_partition_filter: true
+    expiration_days: null
+  clustering:
+    fields:
+    - app_name
```

Comment:
Indeed it would improve performance. From the [GUD](https://gud.telemetry.mozilla.org/#!/explore?graphSize=small&commonScales=true&smoothing=false&mode=explore&startDate=2017-06-17&endDate=2022-04-28&usage=Any%20Firefox%20Desktop%20Activity&metric=all) there is attibution and language. Which would you say are more common @gkabbz?

Reply:
Added country, app_name, attribution_medium and channel to cover the most common searches.